config_defaults_inc.php

```diff
@@ -2787,7 +2787,7 @@
  *
  * @global integer $g_print_reports_threshold
  */
-$g_print_reports_threshold = VIEWER;
+$g_print_reports_threshold = UPDATER;
```

Comment:
I agree that it's safer, but finally it's a weakness of the implementation.
Printing reports is just viewing, so it looks a bit strange to need update rights to view something.
Nevertheless we should keep your change, but it should be mentioned in the release announcement that admins have to set `$g_print_reports_threshold = VIEWER;` to get back the old behavior.

Reply:
I thought about putting DEVELOPER, but in the end I picked UPDATER as the lowest possible role giving a safe default behavior. I am fine with changing that if you prefer.

I am fine with adding a comment in the release notes.

Reply:
> I am fine with adding a comment in the release notes.

Done -> https://mantisbt.org/bugs/roadmap_page.php?version_id=360